include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -141,45 +145,38 @@ struct alignment_configurator
 
     /*!\brief Transformation trait that chooses the correct matrix policy.
      * \tparam config_type The configuration for which to select the correct matrix policy.
-     * \tparam trait_types A template parameter pack with additional traits to augment the selected policy.
+     * \tparam score_t The type used for the score matrix.
+     * \tparam trace_t The type used for the trace matrix.
      */
-    template <typename config_type, typename score_allocator_t, typename trace_allocator_t>
+    template <typename config_type, typename score_t, typename trace_t>
     struct select_matrix_policy
```

Comment:
hm.. we usually do not use verbs in the naming of type traits. Why not `matrix_policy`?

Reply:
Since when ??? 
This is common for most of the type traits in the STL. remove_reference, add_const, enable_if, etc.